### ISSUE TITLE
Make a zoom so people can get output for bounding boxes or points.

### ISSUE THREAD

**Issue by SimonGoring** [PalEON-Project/Shiny#7]
This might be complicated. . .

**Comment by paciorek** [comment]
but cool!

**Comment by SimonGoring** [comment]
Yes, but complicated :)  I think it actually would be somewhat
straightforward as a text edit.  Part of the issue would be transforming
the lat/long and translating it all into gridded Albers outputs.  Anyway,
yes, we could do it, but I think there are some more straightforward things
we could do, I just wanted to put this down the line after we get the
mapping nicer and stuff. . .

**Comment by paciorek** [comment]
Simon had mentioned using leaflet.  This Shiny example (http://shiny.rstudio.com/gallery/superzip-example.html) might be helpful.  I'm not sure what would be involved in dealing with projections in terms of our Albers - presumably the leaflet base map is a different projection and it seems to want lat/lon inputs.

**Comment by SimonGoring** [comment]
Yes, I think leaflet might make things a bit easier, but we'd have to
reproject our data into lat/long.

Simon

**Comment by paciorek** [comment]
At the moment I'm inclined to get what we have looking nice and then
circulate to folks. I think getting a full pan/zoom ability is a bit of a
project on its own and we should figure out who could work on it. Perhaps a
project for a student.

**Comment by SimonGoring** [comment]
Yes, I agree.  This is for later.

**Comment by paciorek** [comment]
Some rough notes from a bit of research I did a few days ago:
ggmap embeds rgooglemaps into ggplot2 - try this for zoom & pan; how
   deal with projection?
http://shiny.rstudio.com/gallery/superzip-example.html - has pan/zoom,
   but what about projection?
map = leaflet() %>% addTiles() %>% setView(-93.65, 42.0285, zoom = 12)
   # will map a leaflet map viewable in browser

**Comment by SimonGoring** [comment]
The geom_tile is mpulling a pre-built representation of the dataset
already.  It's not a problem to just re-project and export everything as
latlong.

All the political boundaries are natively in longlat, using the leaflet
would be nicer :) I can re-run the code so it projects to the longlat
projection.

**Comment by paciorek** [comment]
Well, I made a less fancy zoom that I'll email about momentarily.

I'm not sure what you mean by re-run the code so it projects in longlat.
You mean take my stat model output and reproject or do something with the
raw data. The latter won't help because the stat model is based on using
Albers.

**Comment by SimonGoring** [comment]
I mean take the NCDF mean estimates and sds and then reproject those single
raster layers into a longlat projection.  I'll make a branch and try it out.

We can display in longlat so that we can use leaflet, but export the Albers
because it's the projection everything is based on.  That way the raw
output is statistically sound and the visual display is pretty.
